configuration/pih/htmlforms/hiv/section-hiv-state.xml

```diff
@@ -950,9 +951,9 @@
                                 <obs conceptId="CIEL:1628" style="checkbox" answerConceptId="CIEL:112493"
                                      answerCode="pihcore.syphilisTreatmentPregnantMother" toggle="mom-syph-treat"/>
                             </p>
-                            <p id="mom-syph-treat">
-                                <obs conceptId="CIEL:163526" labelCode="pihcore.treatmentStartDate"/>
-                                <obs conceptId="CIEL:164384" labelCode="pihcore.treatmentEndDate"/>
+                            <p id="mom-syph-treat" class="startDateEndDate">
+                                <obs conceptId="CIEL:163526" labelCode="pihcore.treatmentStartDate" id="startDate"/>
+                                <obs conceptId="CIEL:164384" labelCode="pihcore.treatmentEndDate" id="endDate"/>
```

Comment:
So the trouble with this is that now we've got the same IDs being used multiple times, which is not allowed in HTML. But you could change these to classes, i.e.

```suggestion
                                <obs conceptId="CIEL:163526" labelCode="pihcore.treatmentStartDate" class="startDate"/>
                                <obs conceptId="CIEL:164384" labelCode="pihcore.treatmentEndDate" class="endDate"/>
```